code/game/objects/items/weapons/clown_items.dm

```diff
@@ -105,28 +105,30 @@
 	throw_speed = 3
 	throw_range = 7
 	attack_verb = list("HONKED")
-	var/spam_flag = 0
+	var/spam_flag = FALSE
 	var/honksound = 'sound/items/bikehorn.ogg'
 	var/cooldowntime = 20
 
 /obj/item/weapon/bikehorn/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] solemnly points the horn at [user.p_their()] temple! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	user.visible_message("<span class='suicide'>[user] solemnly points \the [src] at [user.p_their()] temple! It looks like [user.p_theyre()] trying to commit suicide!</span>")
 	playsound(src.loc, honksound, 50, 1)
-	return (BRUTELOSS)
+	return BRUTELOSS
 
 /obj/item/weapon/bikehorn/attack(mob/living/carbon/M, mob/living/carbon/user)
 	if(!spam_flag)
+		spam_flag = TRUE
 		playsound(loc, honksound, 50, 1, -1) //plays instead of tap.ogg!
+		spawn(cooldowntime)

```

Comment:
Please use the `addtimer` and `callback` system to do this instead of using spawns, you can find more information about this [here](https://github.com/HippieStation/HippieStation13/blob/master/code/datums/callback.dm)